acquia/acquia_cms
# ACMS-1652: Fix missing image previews in Site Studio components.

**Motivation**
Fixes ACMS-1652

**Proposed changes**
Added all the missing thumbnails for component pack in site studio.

**Testing steps**

- Install site with site studio 
- go to node/add/page 
- and check if components pack thumbnail is showing.
